resolver-dns/src/test/java/io/netty/resolver/dns/DnsNameResolverTest.java

```diff
@@ -1143,7 +1143,7 @@ public void aAndAAAAQueryShouldTryFirstDnsServerBeforeSecond() throws IOExceptio
                     new TestRecursiveCacheDnsQueryLifecycleObserverFactory();
 
             DnsNameResolverBuilder builder = new DnsNameResolverBuilder(group.next())
-                    .resolvedAddressTypes(ResolvedAddressTypes.IPV6_PREFERRED)
+                    .resolvedAddressTypes(ResolvedAddressTypes.IPV4_ONLY)
```

Comment:
Why this change?

Reply:
@ejona86 it made the test easier to adjust as it also change the number of queries :)

Reply:
Ah. Makes sense.